.github/actions/build_and_push_docker_image/action.yml

```diff
@@ -25,9 +25,6 @@ inputs:
   CA_CRT:
     description: 'K8S CA_CRT'
     required: true
-  CA:
```

Comment:
<img width="697" alt="image" src="https://github.com/binary-com/deriv-app/assets/56330681/6359acf0-97d3-4459-b3c5-fab6ad3ac793">
can you remove the CA in exports here?

Reply:
done